kokkos/kokkos-kernels
# Blas perf_test updates

perf_test/blas:
- Add GFLOP/s output
- Added min/max achieve bandwidth for trmm 
- Add support for separate batch_size option
- Add support for Kokkos::AUTO team_size and vector_len option
- Updated step option to add step size
- Fix batched_serial to use RangePolicy for gemm
- Add support for verify option for gemm
- Add support for batch_size_last_dim_option
- Added batched_serial_simd, batched_serial_simd_blocked, batched_serial_compact_mkl, batched_team_simd, batched_team_simd_blocked test types for gemm
